StortSpelProjekt/Engine/src/Renderer/TextManager.h

```diff
@@ -0,0 +1,49 @@
+#ifndef TEXTMANAGER_H
+#define TEXTMANAGER_H
+
+#include <map>
+#include <vector>
+
+class Window;
+class Texture;
+class Text;
+class Font;
+
+struct TextData;
```

Comment:
Not sure how this works, you should have to include this struct since you are not using a pointer in your m_TextDataMap. Remove this forward Declaration.